logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/receiver/conversation/message/ProteusMessageUnpacker.kt

```diff
@@ -75,7 +76,20 @@ internal class ProteusMessageUnpackerImpl(
             .onFailure {
                 when (it) {
                     is CoreFailure.Unknown -> logger.e("UnknownFailure when processing message: $it", it.rootCause)
-                    is ProteusFailure -> logger.e("ProteusFailure when processing message: ${it.proteusException.code.name}")
+
+                    is ProteusFailure -> {
+                        logger.e("ProteusFailure when processing message: ${it.proteusException.code.name}")
```

Comment:
do we still need this line? Probably not

Reply:
Maybe for the dashboard ? I can remove it and we can update dashboards too

Reply:
Removed :)